### PR TITLE
Fix the Florida Negative Negative_Tests Edge Case

### PR DESCRIPTION
**Pull request by BrettBoval**
Some counties in Florida have erroneously reported test data that was not caught by our test correction filters. This tightens up filter restrictions as a first step towards a more robust pipeline.

https://trello.com/c/88PFNVhy/325-florida-infection-rate-wild-ride

This PR addresses issue #<insert number>

### Please Check
- [ ] Are the [JSON schemas](https://github.com/covid-projections/covid-data-model/tree/master/api/schemas) updated?
- [ ] Are the [api docs](https://github.com/covid-projections/covid-data-model/blob/master/api/README.V1.md) updated?
- [ ] Are tests passing?

### CODE REVIEW

File: pyseir/load_data.py [hunk 1]
```diff
@@ -639,7 +639,8 @@ def load_new_test_data_by_fips(fips, t0, smoothing_tau=5, correction_threshold=5
     df = df.loc[
         (df[CommonFields.POSITIVE_TESTS].notnull())
         & (df[CommonFields.NEGATIVE_TESTS].notnull())
-        & ((df[CommonFields.POSITIVE_TESTS] + df[CommonFields.NEGATIVE_TESTS]) > 0),
+        & (df[CommonFields.NEGATIVE_TESTS] > 0)
+        & (df[CommonFields.POSITIVE_TESTS] > 0),
```
Comment (mikelehen) [inline pyseir/load_data.py:643]: FWIW, positive tests == 0 is normal in some counties that have contained covid (or haven't been hit yet).

But maybe it's okay not to do any test correction in that case.

Comment (BrettBoval) [reply pyseir/load_data.py:643]: The naming convention is confusing. Here CommonFields.POSITIVE_TESTS is cumulative. This is a very permissive filter.
